packages/client/hmi-client/src/services/model-configurations.ts

```diff
@@ -44,6 +44,12 @@ export const updateModelConfiguration = async (config: ModelConfiguration) => {
 		});
 	}
 
+	// If the "default" config is updated we want to update the model as well
+	// because the model as a copy of the data
+	if (config.name === 'Default config') {
+		API.put(`/models/${config.configuration.id}`, config.configuration);
+	}
+
```

Comment:
This should be done on `hmi-server` behind the PUT `/model_configurations/${config.id}` call.

Reply:
This may be a good point for a small discussion.

Putting model-update behind the API-layer makes the interfaces cleaner, but you lose a bit of control because we wouldn't necessarily known when the model itself is updated. 

So the overall question is whether we want to refresh the model independently of the model-configuration updates, or we couple them together. e.g. by calling update model-config, I would known that model-config has updated, but there is also a chance that model is updated ... I need to know if it is necessary to refresh the model-sections because they might be stale.